### ISSUE TITLE
openvas-scanner cann't start

### ISSUE THREAD

**Issue by spacexnasa** [Atomicorp/gvm#19]
lsb_release -a
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: CentOS
Description:    CentOS Linux release 7.6.1810 (Core) 
Release:        7.6.1810
Codename:       Core

systemctl status openvas-scanner.service 
● openvas-scanner.service - OpenVAS Scanner
   Loaded: loaded (/usr/lib/systemd/system/openvas-scanner.service; enabled; vendor preset: disabled)
   Active: failed (Result: start-limit) since Sat 2019-07-13 21:21:28 CST; 6min ago
  Process: 17848 ExecStart=/usr/sbin/openvassd $SCANNER_SOCKET $SCANNER_MODE $SCANNER_GROUP $SCANNER_OWNER (code=exited, status=0/SUCCESS)
 Main PID: 17849 (code=exited, status=1/FAILURE)

Jul 13 21:21:27 localhost.localdomain systemd[1]: Unit openvas-scanner.service entered failed state.
Jul 13 21:21:27 localhost.localdomain systemd[1]: openvas-scanner.service failed.
Jul 13 21:21:28 localhost.localdomain systemd[1]: openvas-scanner.service holdoff time over, scheduling restart.
Jul 13 21:21:28 localhost.localdomain systemd[1]: Stopped OpenVAS Scanner.
Jul 13 21:21:28 localhost.localdomain systemd[1]: start request repeated too quickly for openvas-scanner.service
Jul 13 21:21:28 localhost.localdomain systemd[1]: Failed to start OpenVAS Scanner.
Jul 13 21:21:28 localhost.localdomain systemd[1]: Unit openvas-scanner.service entered failed state.
Jul 13 21:21:28 localhost.localdomain systemd[1]: openvas-scanner.service failed.

**Comment by bpmcdevitt** [comment]
@spacexnasa are you still running into the issue? if so, check the output of `/var/log/gvm/openvassd.log` 

there is a popular guide on the internet that has the user set the redis unix socket to `/tmp/redis.sock`. the guide that i followed that led me to the same issue you were experiencing was located here: [link](https://www.atlantic.net/hipaa-disaster-recovery/how-to-install-openvas-vulnerability-scanner-centos-7/)

this package looks for the redis socket here by default:

```
kb_location=/var/run/redis/redis.sock
db_address=/var/run/redis/redis.sock
```

you can either change the kb_location and db_address values in the /etc/openvas/openvassd.conf to match the socket that you have set in your `/etc/redis.conf` file and then attempt to `systemctl restart openvas-scanner` and it should work

**Comment by ClaudioSousa14** [comment]
> @spacexnasa are you still running into the issue? if so, check the output of `/var/log/gvm/openvassd.log`
> 
> there is a popular guide on the internet that has the user set the redis unix socket to `/tmp/redis.sock`. the guide that i followed that led me to the same issue you were experiencing was located here: [link](https://www.atlantic.net/hipaa-disaster-recovery/how-to-install-openvas-vulnerability-scanner-centos-7/)
> 
> this package looks for the redis socket here by default:
> 
> ```
> kb_location=/var/run/redis/redis.sock
> db_address=/var/run/redis/redis.sock
> ```
> 
> you can either change the kb_location and db_address values in the /etc/openvas/openvassd.conf to match the socket that you have set in your `/etc/redis.conf` file and then attempt to `systemctl restart openvas-scanner` and it should work

I had the same problem and I solved it with your tips.
Really thank you ;)

**Comment by atomicturtle** [comment]
closed, 20.08 is now available
